### ISSUE TITLE
Error updating and adding a user

### ISSUE THREAD

**Issue by alnutile** [alnutile/cms#2]
#5. I was able to enter information for a new user in the /admin/users/new page, but nothing happened when I clicked 'Submit' #4. Similarly, I was able to edit fields for a user, but nothing happened when I clicked 'Submit' #5. What happens if you don't click Active or Admin? What happens if you click both?

**Comment by alnutile** [comment]
@meetinghouse On the Survey I made you noted you could not Edit or Add a user
Nothing happened when you press Submit
Can tell me what browser you where using?
Also please note you must log in first
http://cms.stagingarea.us/login as noted on the top of the doc.
Ideally if you are not logged in the app should direct you back there which is a bug I will fix

**Comment by meetinghouse** [comment]
I see\* rror updating and adding a user #2\* is closed, per Github notes.

Cordially,

John Corbett
Corbett Research Group
"Building Websites That Work, for the Building Artisans"
http://www.restorationtrades.com

Publishers of:
The Custom Building & Restoration Trades Directory
http://www.restorationtradesdirectory.com

Email:  mailto:John@RestorationTrades.comJohn@RestorationTrades.com
Phone: 413-586-4748

**Comment by alnutile** [comment]
John your message got cut? Can you explain. BTW I just noticed
https://www.dropbox.com/s/szdeiz0s7s67i51/Screenshot%202014-04-04%2015.41.31.png
the Edit page still has the old theme. Let me move it over and see why my tests did not catch that.

**Comment by meetinghouse** [comment]
Al,

Not sure of your question. You sent me a link to #2. I visited the link.
The information at the link seemed to be that the issues was closed, so I
replied to your email saying just that, that the issue #2 was closed. As
far as I can see, nothing was cut from my message.

BTW, I visited the link to the screenshot that you furnished and I don't
know if you are asking me to do anything with it or what it means. Is this
the registration UI? If so, I don't understand what the User should do with
the Active toggle. Also, we don't usually associate the login with first
and last name, but I suppose there would be no harm in doing so.

In summary, I'm not sure what function this UI has or if there is anything
actionable that I am supposed to do with it.

Cordially,

John Corbett
Corbett Research Group
"Building Websites That Work, for the Building Artisans"
http://www.restorationtrades.com

Publishers of:
The Custom Building & Restoration Trades Directory
http://www.restorationtradesdirectory.com

Email:  mailto:John@RestorationTrades.comJohn@RestorationTrades.com
Phone: 413-586-4748

**Comment by alnutile** [comment]
Seems like Monday we can do a quick review together of the pages and the
poll and see where it stands.
I will fix the Add new user things shortly so that on Monday (if you have
time) we can take 20 minutes to review it?

Al Nutile
http://alfrednutile.info/about
413.345.5103

**Comment by meetinghouse** [comment]
Al,

I think that would be very useful. I have reviewed all your pending email
docs, tho not sure how effectively. At least I know what I don't know (or
think I do!). Agenda issues are:
- What is finished product? (I sent some notes today w specific
  questions.)
- What has been approved so far? (Stray UI screenshot?)
- What are my current actionable tasks?
- What are the authoritative docs? (Dead links at poll?)

Cordially,

John Corbett
Corbett Research Group
"Building Websites That Work, for the Building Artisans"
http://www.restorationtrades.com

Publishers of:
The Custom Building & Restoration Trades Directory
http://www.restorationtradesdirectory.com

Email:  mailto:John@RestorationTrades.comJohn@RestorationTrades.com
Phone: 413-586-4748
